### DISCUSSION TITLE
The Future of Winglang – Will the Project Stay Alive and Open Source?

### DISCUSSION THREAD

**Discussion by Galz648** [winglang/wing#7225] [General]
Hello, Winglang community! 👋

As someone interested in contributing to this amazing project, I’m curious about its future:

Will Winglang remain open source in the long term?
What are the key areas where contributors like me can make the biggest impact?
Looking forward to hearing your thoughts and helping Winglang thrive!

**Comment by ShaiBer** [comment]
Hi @Galz648 , thanks for contributing!

Yes, we plan to keep Winglang open in the long term, as almost all programming languages are.
I think the key areas to contribute depend on what interests you. You can pick something from the "[good first issue](https://github.com/winglang/wing/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)" label, or you can set up time to chat with @eladb and me to pick something together

**Reply by Galz648** [reply]
I wonder if we could discuss an area in the wing project, that a developer at my level could contribute to, would love guidance and assistance along the way.

**Reply by eagleusb** [reply]
@ShaiBer is it still the case? The project seems abandoned.
